radionets/evaluation/plotting.py

```diff
@@ -629,13 +629,13 @@ def plot_box(ax, num_boxes, corners):
 
 
 def histogram_ms_ssim(msssim, out_path, plot_format="png"):
+    msssim = msssim.numpy()
     mean = np.mean(msssim)
     std = np.std(msssim, ddof=1)
     fig, (ax1) = plt.subplots(1, figsize=(6, 4))
-    bins = np.arange(msssim.min(), 1 + 0.01, 0.01)
     ax1.hist(
         msssim,
-        bins=bins,
+        bins=30,
```

Comment:
why not leave `bins` here and make it a function argument?

Reply:
Done